repair_refurbish/models/repair.py

```diff
@@ -50,7 +50,17 @@ def _get_refurbish_stock_move_dict(self):
             'location_id': self.location_dest_id.id,
             'location_dest_id': self.refurbish_location_dest_id.id,
             'restrict_lot_id': self.refurbish_lot_id.id,
```

Comment:
I guess this line can be deleted.

Reply:
@grindtildeath these lines haven't disturb anything at all. so we are good to leave as it is. Please correct me. `restrict_lot_id` field is remove from Odoo 12, so we can remove that.

Reply:
@bodedra Yes, I only meant to remove the `restrict_lot_id` one, the other lines must of course stay there.
Actually this line gives a warning in server logs:
`WARNING oca_manufacture odoo.models: stock.move.create() with unknown fields: restrict_lot_id `